README.md

```diff
@@ -41,7 +41,7 @@ default['supermarket_omnibus']['package_version'] = '1.2.3'
 default['supermarket_omnibus']['package_repo'] = 'current'
 
 # OR, specify a Supermarket package explicitly from a location of your choosing
-default['supermarket_omnibus']['package_url'] = 'http://bit.ly/98K8eH'
```

Comment:
That was a long con!
![hats off to you](https://media.tenor.com/images/cca860bf3e19f9c21b91df0552771185/tenor.gif)